src/api/json/catalog.json

```diff
@@ -437,6 +437,22 @@
       "description": "Schema for the representation of 3D city models",
       "url": "https://raw.githubusercontent.com/cityjson/specs/1.0.1/schemas/cityjson.min.schema.json"
     },
+    {
+      "name": "Helm Chart.yaml",
+      "description": "The Chart.yaml file is required for a chart.",
+      "fileMatch": [
+        "Chart.yaml"
```

Comment:
This seems pretty generic. Are all Helm charts using that filename or could they use any file name?

Reply:
All Helm charts use this filename. `Chart.yaml` for a Helm chart is similar to `package.json` for an npm project. The file structure of a Helm chart can be found [here](https://helm.sh/docs/topics/charts/#the-chart-file-structure).